src/openrct2-ui/windows/LoadSave.cpp

```diff
@@ -349,8 +349,8 @@ static bool Browse(bool isSave, char* path, size_t pathSize)
     switch (_type & 0x0E)
     {
         case LOADSAVETYPE_GAME:
-            extension = ".sv6";
-            fileType = FILE_EXTENSION_SV6;
+            extension = ".park";
```

Comment:
Shouldn’t this list _all_ the types that OpenRCT2 can open?

(May be done in a later PR.)

Reply:
That's handled by a different part of this